curvefs/src/metaserver/storage/storage.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: Curve
+ * Date: 2022-02-14
+ * Author: Jingli Chen (Wine93)
+ */
+
+#include <memory>
+
+#include "curvefs/src/metaserver/storage/storage.h"
+#include "curvefs/src/metaserver/storage/memory_storage.h"
+
+namespace curvefs {
+namespace metaserver {
+namespace storage {
+
+using ::curvefs::metaserver::storage::KVStorage;
+
+static std::shared_ptr<KVStorage> kKVStorage;
+
+bool InitStorage(StorageOptions options) {
+    kKVStorage = std::make_shared<MemoryStorage>(options);
+    return true;
```

Comment:
it always return true, can return void instead

Reply:
> it always return true, can return void instead

agree!

Reply:
> it always return true, can return void instead

done.